File: totosuki/ABC/B/016.py
A, B, C = map(int, input().split())
is_plus = False
is_minus = False
if A+B == C:
  is_plus = True
if A-B == C:
  is_minus = True

if is_plus and is_minus:
  print("?")
elif is_plus:
  print("+")
elif is_minus:
  print("-")
else:
  print("!")

File: OCH_Tao/ABC/A/016.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").split("\n");
const MD = input[0].split(" ");
const M = parseInt(MD[0], 10);
const D = parseInt(MD[1], 10);
if (M % D === 0) {
  console.log("YES");
} else {
  console.log("NO");
}